odoo/addons/base/models/ir_sequence.py

```diff
@@ -271,7 +271,7 @@ def next_by_code(self, sequence_code, sequence_date=None):
         seq_ids = self.search([('code', '=', sequence_code), ('company_id', 'in', [company_id, False])], order='company_id')
         if not seq_ids:
             _logger.debug("No ir.sequence has been found for code '%s'. Please make sure a sequence is set for current company." % sequence_code)
-            return False
+            raise UserError(_("No ir.sequence has been found for code '%s'. Please make sure a sequence is set for current company." % sequence_code))
```

Comment:
Please rewrite according to [Odoo Guideline](https://www.odoo.com/documentation/13.0/reference/guidelines.html#use-translation-method-correctly)